docs/tutorials/portal/103/portal-103-4.rst

```diff
@@ -109,7 +112,6 @@ which identifies an LSST visit.
 
 **6. Review the three-table join results.**
 The join of ``Object`` to ``ForcedSource`` is one-to-many, and the join of ``ForcedSource`` to ``CcdVisit`` is many-to-one.
```

Comment:
The existing statement about clicking "Coverage" tab to view the coverage chart was removed, but that's exactly the statement this ticket was aiming to ensure existed.